crates/lib/kajiya/src/renderers/rtr.rs

```diff
@@ -184,6 +184,41 @@ impl RtrRenderer {
             temporal2_tex: &mut self.temporal2_tex,
         }
     }
+
+    pub fn create_dummy_output(
+        &mut self,
+        rg: &mut rg::TemporalRenderGraph,
+        gbuffer_depth: &GbufferDepth,
+    ) -> TracedRtr {
+        let gbuffer_desc = gbuffer_depth.gbuffer.desc();
+
+        let mut resolved_tex = rg.create(
+            gbuffer_depth
+                .gbuffer
+                .desc()
+                .usage(vk::ImageUsageFlags::empty())
+                .format(vk::Format::R16G16B16A16_SFLOAT),
+        );
+
+        let (temporal_output_tex, history_tex) = self
+            .temporal_tex
+            .get_output_and_history(rg, Self::temporal_tex_desc(gbuffer_desc.extent_2d()));
+
+        let (mut ray_len_output_tex, ray_len_history_tex) =
+            self.ray_len_tex.get_output_and_history(
+                rg,
+                ImageDesc::new_2d(vk::Format::R16G16_SFLOAT, gbuffer_desc.extent_2d())
+                    .usage(vk::ImageUsageFlags::SAMPLED | vk::ImageUsageFlags::STORAGE),
+            );
```

Comment:
Technically those could also be `R8G8B8A8_UNORM`, but it would also be possible to plug in SSR here, so not a big deal.